apps/design-system-docs/docs/components/packages/wp-components/icon-link/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button } from '@wordpress/components';
+import { IconFigma } from './icon-figma';
+import { IconStorybook } from './icon-storybook';
+
+export const IconLink = ( { href, type }: { href?: string; type: 'figma' | 'storybook' } ) => {
+	if ( ! href ) {
+		return null;
+	}
```

Comment:
Nit: IMO, it'd make more sense to have `WPComponentsTable` avoid rendering `IconLink` if the link is empty, rather than handling that here.

Reply:
Fixed in 5280683 👍